src/angularjs/src/app/places/list/place-list.html

```diff
@@ -75,7 +74,7 @@ <h2 class="sidebar-title">Place Search</h2>
 
         <section>
             <!-- Duplicate .result div for each result. -->
-            <div class="result" ng-repeat="neighborhood in placeList.places">
+            <div class="result" ng-repeat="neighborhood in placeList.places track by neighborhood.uuid">
```

Comment:
This fixes a bug where the list would re-render every item in the ng-repeat when the list of places was changed. Now it properly only re-renders the diff.